x/harvest/types/params.go

```diff
@@ -223,22 +223,22 @@ type Multipliers []Multiplier
 
 // BorrowLimit enforces restrictions on a money market
 type BorrowLimit struct {
-	MaximumLimit sdk.Int `json:"maximum_limit" yaml:"maximum_limit"`
-	LoanToValue  sdk.Dec `json:"loan_to_value" yaml:"loan_to_value"`
+	MaximumLimitUSD sdk.Dec `json:"maximum_limit_usd" yaml:"maximum_limit_usd"`
```

Comment:
This should just be in native units, not usd. Further, the max borrow limit should be an optional flag. Something like

```
type BorrowLimit struct {
        HasMaxLimit bool `json:"has_max_limit" yaml:"has_max_limit"`
	MaximumLimitUSD sdk.Int `json:"maximum_limit_usd" yaml:"maximum_limit_usd"`
	LoanToValue     sdk.Dec `json:"loan_to_value" yaml:"loan_to_value"`
}

Reply:
Added in https://github.com/Kava-Labs/kava/pull/715/commits/777fc3e26d0db82666294056b9b923a1f0061f92